pkg/controller/certapi/certificate.go

```diff
@@ -45,17 +44,8 @@ func (c *Controller) HandleCertificate() http.Handler {
 			return
 		}
 
-		// This is a non terminal error, as we're only using the realm for stats.
-		realm, err := authApp.Realm(c.db)
-		if err != nil {
-			c.logger.Errorf("unable to load realm", "error", err)
-		} else {
-			ctx, err = tag.New(ctx,
```

Comment:
I think this was actually a subtle bug that we just never hit... If `tag.New` has an error, I think the returned context is nil.